### PR TITLE
Clean and fix ZitaReverb

### PR DESCRIPTION
**Pull request by psmokotnin**
Fixes for ZitaReverb:
- params check

- default initialized values

### CODE REVIEW

**Comment by RobFog** [comment]
I reckon this should address https://trello.com/c/oAUqTd41/48-strange-effects-stereo-issue-causes-poor-playback-quality

**Comment by RomanPudashkin** [comment]
Could you please explain the reason why we can't just revert the previous fix
https://github.com/musescore/MuseScore/pull/6832

I'm afraid that this fix will be a reason for a new bunch of hard reproduced bugs

**Comment by psmokotnin** [comment]
@RomanPudashkin then you'll still have a possibility of receiving the wrong numbers in reverb's parameters.

**Comment by RomanPudashkin** [comment]
> @RomanPudashkin then you'll still have a possibility of receiving the wrong numbers in reverb's parameters.

Can you reproduce this situation or this is an assumption?

**Comment by psmokotnin** [comment]
@RomanPudashkin when bad value (out of range) stored in the score file:
image

File: effects/zita1/zita.h [hunk 1]
```diff
@@ -201,28 +218,27 @@ class ZitaReverb : public Effect
       {
       Q_OBJECT
 
-      float   _fsamp = 0.f;
+      float   _fsamp = 1.f;
```
Comment (psmokotnin) [inline effects/zita1/zita.h:221]: @RomanPudashkin another interesting moment. If we keep `0.f;` and object won't be initialized, than will catch division by zero;

**Pull request merged**